projects/handlefi/index.js

```diff
@@ -9,7 +9,9 @@ const abi = require("./abi.json");
 const transformArbitrumAddress = addr => `arbitrum:${addr}`
 const treasuryContract = "0x5710B75A0aA37f4Da939A61bb53c519296627994"
 const WETH_arbitrum = "0x82af49447d8a07e3bd95bd0d56f35241523fbab1"
-const treasuryTokens = [WETH_arbitrum]
+const FOREX_arbitrum = "0xDb298285FE4C5410B05390cA80e8Fbe9DE1F259B"
+const treasuryTokens = [WETH_arbitrum,FOREX_arbitrum]
+const perpsVault = "0x1785e8491e7e9d771b2A6E9E389c25265F06326A"
```

Comment:
what is this vault, I dont see it being used anywhere

Reply:
this is the vault for FX perps liquidity

app.handle.fi/trade

https://arbiscan.io/address/0x1785e8491e7e9d771b2a6e9e389c25265f06326a

Reply:
yes, the vault address is legit but I dont see it being used in the code here, maybe you forgot to add it to tvl?

Reply:
yep, you are correct - it has not been added! did you want me to remove and wait until another PR?

Reply:
no, it is fine if it is intentional.